zhaquirks/tuya/ts001x.py

```diff
@@ -308,6 +308,104 @@ class TuyaTripleNoNeutralSwitch(TuyaSwitch):
     }
 
 
+class TuyaTripleNoNeutralSwitchV2(EnchantedDevice, TuyaSwitch):
+    """Tuya 3 gang no neutral light switch (v2)."""
```

Comment:
I want to propose call as variants (opposite to v2 that seems to be a newer version).
```python
class TuyaTripleNoNeutralSwitchVar03(EnchantedDevice, TuyaSwitch):
    """Tuya 3 gang no neutral light switch (variant 03)."""
```

Reply:
makes sense, I took it a step further, what do you think?